.github/workflows/presubmit.yml

```diff
@@ -259,6 +259,8 @@ jobs:
         run: |
           ${{ env.testbindir }}/simple_test
           ./tests/perfetto/check-perfetto-trace.sh ${CLVK_PERFETTO_TRACE_DEST} ./tests/perfetto/simple_test-expectation.txt
+          ${{ env.testbindir}}/api_tests --gtest_filter=WithCommandQueue.EnqueueTooManyCommandsWithRetry
+          ./tests/perfetto/check-perfetto-trace.sh ${CLVK_PERFETTO_TRACE_DEST} ./tests/perfetto/api_tests.EnqueueTooManyCommandWithRetry-expectation.txt
```

Comment:
It's nice to be able to check that we're hitting specific functions but I am concerned about how scalable/maintainable these tests will be if we start adding a few of them. It's not a problem right now but we need to keep an eye on this.